packages/client/src/client/notifications.openfin.ts

```diff
@@ -215,6 +218,48 @@ const sendCreditQuoteNotification = (quote: PricedQuoteDetails) => {
   create(notificationOptions)
 }
 
+const sendRFQCreatedConfirmationNotification = (
+  rfqRequestConfirmation: ConfirmCreatedCreditRfq,
+) => {
+  const { title, rfqDetails } = processRFQRequestConfirmation(
+    rfqRequestConfirmation,
+  )
+
+  const notificationOptions: TemplateCustom = {
+    template: "custom",
+    templateOptions: createNotificationTemplate(
+      rfqRequestConfirmation.request.direction,
+    ),
+    templateData: {
+      messageTradeDirection:
+        rfqRequestConfirmation.request.direction.toUpperCase(),
+      messageTradeDetails: rfqDetails,
+    },
+
+    icon: creditIconUrl,
+    category: "RFQ Created",
+    title,
+    indicator: {
+      text: "new rfq",
+      color: IndicatorColor.GRAY,
+    },
+    buttons: [
```

Comment:
you have added the View RFQ button here .. but does it work? 🤔
there doesn't seem to be the associated logic in `registerCreditCreatedNotifications` as there is in `registerCreditQuoteNotifications`
this is another potential advantage to aggregating the Credit registrations ...

Reply:
Yes, it works, it works just like the other notification when we accept or reject, it highlights the newly created RFQ.

Reply:
ah, it was a side effect of running the other registration .. cannot rely on that in case we modify the other registration